src/main/java/io/stargate/sgv2/jsonapi/service/resolver/VectorizeConfigValidator.java

```diff
@@ -344,6 +356,18 @@ private Integer validateModelAndDimension(
                         "Model name '%s' for provider '%s' is not supported",
                         userConfig.modelName(), userConfig.provider()));
 
+    // validate model support
+    if (model.modelSupport().status() != ModelSupport.SupportStatus.SUPPORTED) {
+      throw SchemaException.Code.UNSUPPORTED_PROVIDER_MODEL.get(
+          Map.of(
+              "model",
+              userConfig.modelName(),
+              "modelStatus",
+              model.modelSupport().status().name(),
+              "message",
+              model.modelSupport().message().orElse("The model is not supported.")));
```

Comment:
Same as earlier:  maybe

"The model is no longer supported (status: END_OF_LIFE)"